source/common/http/http2/codec_impl.cc

```diff
@@ -1692,6 +1699,58 @@ int64_t ConnectionImpl::Http2Visitor::OnReadyToSend(absl::string_view serialized
                              serialized.size());
 }
 
+ConnectionImpl::Http2Visitor::DataFrameHeaderInfo
+ConnectionImpl::Http2Visitor::OnReadyToSendDataForStream(Http2StreamId stream_id,
+                                                         size_t max_length) {
+  StreamImpl* stream = connection_->getStream(stream_id);
+  if (stream == nullptr) {
+    return {-1, false, false};
```

Comment:
optional, here and below: Could add `/*name=*/` for the bools.

Reply:
Done, thanks!